src/waveform/guitick.cpp

```diff
@@ -23,18 +20,12 @@ GuiTick::~GuiTick() {
 // this is called from the VSyncThread
 // with the configured waveform frame rate
 void GuiTick::process() {
-    qint64 elapsedNs = m_cpuTimer.restart();
-    double elapsedS = elapsedNs / 1000000000.0;
-    m_cpuTimeLastTick += elapsedS;
-    m_pCOGuiTickTime->set(m_cpuTimeLastTick);
+    mixxx::Duration elapsed = m_cpuTimer.restart();
+    m_cpuTimeLastTickSeconds += elapsed.toDoubleSeconds();
```

Comment:
We should convert m_cpuTimeLastTickSeconds into mixxx::Duration as well to prevent rounding issues